--- gshogi-7c4bd90199c5bda61984347ea68c32521e82a637/setup.py	2020-11-18 19:47:27.000000000 +0900
+++ gshogi-7c4bd90199c5bda61984347ea68c32521e82a637-new/setup.py	2025-08-31 13:54:13.241055450 +0900
@@ -1,5 +1,3 @@
-import ez_setup
-ez_setup.use_setuptools()
 from setuptools import setup, Extension
 
 import os
@@ -21,20 +19,13 @@
         ]
 
 if os.name == "nt":
-    # windows
     macros.append(("HASHFILE", "1"))
-    data_files=[]
 else:
-    # linux
     macros.append(("HAVE_UNISTD_H", "1"))
     macros.append(("HASHFILE", "\"data/gnushogi.hsh\""))
-    data_files=[
-      (sys.prefix+'/share/applications',['gshogi.desktop']),
-      (sys.prefix+'/share/pixmaps', ['gshogi.png'])]
 
 package_data_list = ["data/opening.bbk"]
 
-# translations
 if shutil.which("msgfmt") is None:
     print("msgfmt not found. Translations will not be built")
 else:
@@ -75,7 +66,6 @@
     define_macros=macros,
 )
 
-
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
@@ -91,18 +81,15 @@
       platforms=["Linux"],
       license="GPLv3+",
       zip_safe=False,
-
       packages=["gshogi"],
       package_data={
           "gshogi": package_data_list,
       },
-      data_files=data_files,
       entry_points={
           "gui_scripts": [
               "gshogi = gshogi.gshogi:run",
           ]
       },
-
       classifiers=[
           "Development Status :: 4 - Beta",
           "Environment :: X11 Applications :: GTK",
@@ -113,5 +100,5 @@
           "Programming Language :: C",
           "Topic :: Games/Entertainment :: Board Games",
           ],
-
       )
+
